website/client/src/pages/settings/components/yourBalance.vue

```diff
@@ -28,6 +29,9 @@ export default {
     amountNeeded: {
       type: Number,
     },
+    neededCurrencyOnly: {
+      type: Boolean,
+    },
```

Comment:
On second look re the above comment ... we don't actually use `yourBalance` anywhere else on the client :thinking: In which case the only change necessary is that we can remove the `neededCurrencyOnly` property definition from here. It's currently unused.